tests/transform/test_gene_info.py

```diff
@@ -70,6 +70,11 @@
                        or is_tep values are assumed to mean "False". These two fields must have boolean values if the
```

Comment:
Make sure to update the comment to say "This transform requires **13** different input datasets" now that we've added one!